recipe/meta.yaml

```diff
@@ -29,10 +29,10 @@ requirements:
     - lomap2 >=2.1.0
     - networkx
     - numpy <1.24
-    - openff-toolkit ==0.13
+    - openff-toolkit >=0.13,<0.14
```

Comment:
Isn't this the same? I thought ==0.13 would allow any 0.13.x release?

Reply:
Almost! See https://conda.io/projects/conda-build/en/latest/resources/package-spec.html#examples

If you want a fuzzy match like that, you need only one `=`, as written that will match `0.13` or `0.13.0` or `0.13.0.0` etc

I dislike that syntax, it is a huge foot gun, so I like the explicitness of a range. I could change it to `openff-toolkit =0.13` if you want